lms/static/js/views/fields.js

```diff
@@ -495,6 +495,207 @@
             }
         });
 
+        FieldViews.ImageFieldView = FieldViews.FieldView.extend({
+
+            fieldType: 'image',
+
+            templateSelector: '#field_image-tpl',
+            uploadButtonSelector: '.upload-button-input',
+
+            titleAdd: gettext("upload an image"),
+            titleEdit: gettext("change image"),
+            titleRemove: gettext("remove"),
```

Comment:
![screen shot 2015-04-13 at 7 31 31 pm](https://cloud.githubusercontent.com/assets/6767924/7117696/eae1a93a-e213-11e4-8ee0-f3b5e272d4bd.png)

@srpearce Actually we don't have enough space to accommodate the whole string. 
Is the whole string necessary or just remove is enough?